src/zope/interface/tests/test_odd_declarations.py

```diff
@@ -258,7 +258,7 @@ class C(A, B):
         c.c
 
         import sys
-        if sys.version[0] == '2': # This test only makes sense under Python 2.x
+        if sys.version_info[0] < 3: # This test only makes sense under Python 2.x
```

Comment:
Yeah, it probably is :) It started out as a copy of the doctest in odd.py, I just fixed this when I was globally fixing the same thing in other files.